arch/risc-v/src/common/riscv_pmp.c

```diff
@@ -100,6 +100,32 @@ typedef struct pmp_entry_s pmp_entry_t;
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: log2ceil
+ *
+ * Description:
+ *   Calculate the up-rounded power-of-two for input.
+ *
+ * Input Parameters:
+ *   size - The size of the PMP region.
+ *
+ * Returned Value:
+ *   Power-of-two for argument, rounded up.
+ *
+ ****************************************************************************/
+
+static uintptr_t log2ceil(uintptr_t size)
```

Comment:
can use LOG2_CEIL from https://github.com/apache/nuttx/blob/master/include/nuttx/lib/math32.h#L59

Reply:
Hmm, was looking for this but could not find it. Maybe I'll clean it up at some point.